test/turbopack-tests-manifest.json

```diff
@@ -13408,14 +13408,15 @@
       "Basics default setting with react 18 dev should contain dynamicIds in next data for dynamic imports",
       "Basics default setting with react 18 dev should only render once in SSR",
       "Basics default setting with react 18 dev useId() values should match on hydration",
-      "Concurrent mode in the experimental-edge runtime dev <RouteAnnouncer /> should not have the initial route announced",
       "Concurrent mode in the experimental-edge runtime dev flushes styled-jsx styles as the page renders",
       "Concurrent mode in the experimental-edge runtime dev should not have invalid config warning",
       "Concurrent mode in the nodejs runtime dev <RouteAnnouncer /> should not have the initial route announced",
       "Concurrent mode in the nodejs runtime dev flushes styled-jsx styles as the page renders",
       "Concurrent mode in the nodejs runtime dev should not have invalid config warning"
     ],
-    "failed": [],
+    "failed": [
+      "Concurrent mode in the experimental-edge runtime dev <RouteAnnouncer /> should not have the initial route announced"
```

Comment:
this test never passed actually: 

the assertion is to check if rendered page does _not_ have a string (https://github.com/vercel/next.js/blob/7aef93dd14d5474bad8afd4be02fda13a44ad953/test/integration/react-18/test/index.test.js#L26), so the current rendered page passes since it is a error overlay with panic message only.